_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,445 @@
+---
+layout: default
+title: "Open First Strategy"
+ref: open-first-strategy
+lang: en
+status: In Progress;primary
+sections: Strategies
+permalink: /open-first-strategy.html
+---
+<!-- the below cSpell statement says to ignore any text between HTML tags. E.g. it will ignore "th rowspan='2'" in this string: <th rowspan='2'> -->
+<!-- cSpell:ignoreRegExp /\<[^\>]+\>/ -->
+
+1. Introduction
+2. Vision
+3. Commitment
+4. Goals and outcomes
+5. Governing principles
+6. Implementing the strategy
+
+## 1. Introduction
+
+The Government of Canada (GC) aims to invigorate Canada's unique social market economy, promote competition and encourage SMEs — our innovators and entrepreneurs.
+We want to bring Canada's people together in an inclusive, open approach, to find new opportunities and transition to an inclusive, better digital environment that is ready for the realities of today's global economy.
+
+In all of this, open standards and open source software (OSS) have a role to play.
+
+OSS is present everywhere.
```

Comment:
In this digital age, it is a pervasive technology that is embedded into all facets of our daily lives.  It keeps our houses warm, our identities safe, and our cars on the roads.